### PR TITLE
Start cleaning up node API Dokka comments

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

**Comment by rnicoll** [comment]
Please do let me know if I've misunderstood anything. For context on how this currently looks, please see https://docs.corda.net/head/api/javadoc/

File: node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt [hunk 1]
```diff
@@ -20,52 +12,59 @@ import org.apache.activemq.artemis.reader.MessageUtil
...
     private val TAG_FIELD_NAME = "tag"
     private val RPC_ID_FIELD_NAME = "rpc-id"
     private val OBSERVABLE_ID_FIELD_NAME = "observable-id"
     private val METHOD_NAME_FIELD_NAME = "method-name"
 
-    val RPC_SERVER_QUEUE_NAME = "rpc.server"
-    val RPC_CLIENT_QUEUE_NAME_PREFIX = "rpc.client"
-    val RPC_CLIENT_BINDING_REMOVALS = "rpc.clientqueueremovals"
-    val RPC_CLIENT_BINDING_ADDITIONS = "rpc.clientqueueadditions"
+    /** Name of the Artemis queue on which the node receives RPC requests (as [ClientToServer.RpcRequest]). */
```
Comment (exFalso) [inline node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt]: "node" here and in the rest of the comments is not really accurate, the RPC framework does *not* rely on being part of the node, in fact rpc integ tests create another kind of RPC servers. I think just a search and replace from node -> server would be fine

File: node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt [hunk 2]
```diff
@@ -20,52 +12,59 @@ import org.apache.activemq.artemis.reader.MessageUtil
...
+    const val RPC_SERVER_QUEUE_NAME = "rpc.server"
+    /**
+     * Prefix to Artemis queue names used by clients to receive communication back a node. The full queue name
+     * should be of the form "rpc.client.&lt;username&gt;.&lt;nonce&gt;".
```
Comment (exFalso) [inline node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt]: i think cthulhu took over your keyboard for a bit there

File: node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt [hunk 3]
```diff
@@ -83,12 +82,23 @@ object RPCApi {
...
         private enum class Tag {
             RPC_REQUEST,
             OBSERVABLES_CLOSED
         }
 
+        /**
+         * Request to a node to trigger the specified method with the provided arguments.
+         *
+         * @param clientAddress return address to contact the client at.
+         * @param id a unique ID for the request, which the node will used to identify its response with.
```
Comment (exFalso) [inline node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt]: used -> use

File: node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt (hunk 2 above)
Comment (rnicoll) [reply node-api/src/main/kotlin/net/corda/nodeapi/RPCApi.kt]: It's HTML entities, renders to `rpc.client.<username>.<nonce>` in the actual documentation.

**Comment by rnicoll** [comment]
Addressed comments and rebased, over to you @exFalso

**Pull request merged**